solr/core/src/test/org/apache/solr/search/facet/SpatialHeatmapFacetsTest.java

```diff
@@ -438,8 +436,7 @@ public void testJsonFacets() throws Exception {
                       + " } "
                       + "}"));
       {
-        NamedList<Object> entries1 = response.getResponse();
-        final NamedList<?> q1Res = (NamedList<?>) entries1._get(List.of("facets", "q1"), null);
+          final NamedList<?> q1Res = (NamedList<?>) response.getResponse()._get(List.of("facets", "q1"), null);
```

Comment:
remember "gw tidy", I see here and elsewhere.  I think you can configure IntelliJ better so that basic indentation like this would not be incorrect so often.  I have an idea that I'll raise elsewhere that should reduce this annoyance.